include/h2o.h

```diff
@@ -1909,10 +1932,17 @@ enum {
     H2O_HEADERS_CMD_UNSET       /* removes the named header(s) */
 };
 
+typedef enum h2o_headers_command_when {
+    H2O_HEADERS_CMD_WHEN_FINAL,
+    H2O_HEADERS_CMD_WHEN_EARLY,
+    H2O_HEADERS_CMD_WHEN_ALL,
+} h2o_headers_command_when_t;
```

Comment:
What's the default value? I would suggest moving the default value to the top so that it would be assigned zero.

Reply:
Yes, default value is `H2O_HEADERS_CMD_WHEN_FINAL`